doc/api/deprecations.md

```diff
@@ -2051,12 +2051,15 @@ expose values under these names.
 ### DEP0109: `http`, `https`, and `tls` support for invalid URLs
 <!-- YAML
 changes:
+  - version: v15.5.1
```

Comment:
Use the placeholder instead of a specific version number (it will be change by the releaser)
```suggestion
  - version: REPLACEME
```